cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -277,3 +283,10 @@ def _check_single_assignment(self):
                         "Server, Server Template, or Plan Line Action."
                     )
                 )
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            current_datetime = datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
```

Comment:
use fields.Datetime.now() and see my comments above

Reply:
I tested this method, because it rounds the values ​​I can't use it
```
datetime.datetime.now()
2024-10-31 19:54:14.310060

fields.Datetime.now() 
2024-10-31 19:54:14.000000
```

Reply:
why? add comments to code, I don't understand